SpatialGDK/Source/SpatialGDK/Private/Utils/Interest/ResultTypes.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/Interest/ResultTypes.h"
+
+#include "UObject/UObjectIterator.h"
+
+#include "SpatialGDKSettings.h"
+
+namespace SpatialGDK
+{
+
+ResultType ResultTypes::CreateClientNonAuthInterestResultType(USpatialClassInfoManager* ClassInfoManager)
+{
+	ResultType ClientNonAuthResultType;
+
+	// Add the required unreal components
+	ClientNonAuthResultType.Append(SpatialConstants::REQUIRED_COMPONENTS_FOR_NON_AUTH_CLIENT_INTEREST);
+
+	// Add all data components- clients don't need to see handover or owner only components on other entities.
+	ClientNonAuthResultType.Append(ClassInfoManager->GetComponentIdsForComponentType(ESchemaComponentType::SCHEMA_Data));
+
+	// In direct disagreement with the above comment, we add the owner only components as well.
```

Comment:
I dislike this comment knowing that it is in direct disagreement. Can be confusing. I would combine these two comments into one larger comment and explain why.

Reply:
This was to separate the temporary comment (the TODO) from the comment that is supposed to be here above. Do you think moving the TODO to the top of this comment would make this clearer? Or perhaps rewording it?

Reply:
I think rewording it would help

Reply:
reworded to make it obvious that this is a temporary thing that should be fixed (I hope)